server/src/main/java/org/opensearch/search/profile/ProfileShardResult.java

```diff
@@ -49,9 +50,20 @@
 
     private final AggregationProfileShardResult aggProfileShardResult;
 
-    public ProfileShardResult(List<QueryProfileShardResult> queryProfileResults, AggregationProfileShardResult aggProfileShardResult) {
+    private long inboundNetworkTime;
+
+    private long outboundNetworkTime;
+
+    public ProfileShardResult(
+        List<QueryProfileShardResult> queryProfileResults,
+        AggregationProfileShardResult aggProfileShardResult,
+        long inboundNetworkTime,
+        long outboundNetworkTime
```

Comment:
I think we can move these into a separate class that can be used in both the request and result. This will also make it easier to add new stats.

Reply:
Done - moved into a separate class - NetworkTime.java